src/Data/Nat/Primality/Factorization.agda

```diff
@@ -0,0 +1,125 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Prime factorization of naturla numbers and its properties
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Nat.Primality.Factorization where
+
+open import Data.Empty
+open import Data.Nat.Base
+open import Data.Nat.Divisibility
+open import Data.Nat.Properties
+open import Data.Nat.Induction
+open import Data.Nat.Primality
+open import Data.Nat.Rough
+open import Data.Product as Π
+open import Data.List.Base
+open import Data.List.Relation.Unary.All as All
+open import Data.List.Relation.Binary.Permutation.Propositional as ↭
+open import Data.List.Relation.Binary.Permutation.Propositional.Properties
+open import Data.Sum.Base
+open import Function.Base
+open import Relation.Nullary.Decidable
+open import Relation.Nullary.Negation
+open import Relation.Binary.PropositionalEquality as ≡
+
+-- Core definition
+------------------------------------------------------------------------
+
+record Factorization (n : ℕ) : Set where
```

Comment:
Grepping for `zation` vs. `sation`, we seem to be using the British spelling more often (although not systematically).
I would be in favour of making it the official spelling.